@ecomplus/storefront-template/content/widgets/martan.json

```
@@ -0,0 +1,21 @@
+{
+  "pkg": "@ecomplus/widget-martan",
+  "active": true,
+  "options": {
+    "martanStoreId": 1,
+    "martanWidgetKey": "WK_OTQ4ZWY4OTktYjUwZS01ODcwLWE4YzAtMTE4ODFkY2M2NDc0",
```

```suggestion
    "martanStoreId": 0,
    "martanWidgetKey": "",
```
Inicializando como widget novo